aiohttp_sendgrid/sendgrid.py

```diff
@@ -1,6 +1,5 @@
 import aiohttp
 import os
-import sendgrid
```

Comment:
I removed this because it wasn't referenced in setup.py and didn't seem to be used.